ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/event-search/EventSearch.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import { useState } from 'react';
+import { Button } from '@inngest/components/Button';
+import { useClient } from 'urql';
+
+import Input from '@/components/Forms/Input';
+import { useEnvironment } from '@/queries';
+import { EventTable } from './EventTable';
+import { searchEvents } from './searchEvents';
+import type { Event } from './types';
+
+const day = 1000 * 60 * 60 * 24;
```

Comment:
A day in milliseconds?

Reply:
Yea, just thought that was a little easier to see than plopping `lowerTime: new Date(Date.now() - 3 * 1000 * 60 * 60 * 24)`. We'll replace this date stuff with a real date picker as we work on this feature